test/physics/em/BetheHeitler.test.cc

```diff
@@ -58,11 +58,11 @@ class BetheHeitlerInteractorTest : public celeritas_test::InteractorHostTestBase
               stable},
              {"gamma", pdg::gamma(), zero, zero, stable}});
         const auto& params    = this->particle_params();
```

Comment:
I prefer not to use `const auto&` binding to a shared pointer -- if you don't know ahead of time, you'll be very confused as how to use it.  For this case, just do:
```c++
const auto& params    = *this->particle_params();
```
and the code below can be reverted to what's on master. I recommend doing the same in the other test code that's changed (sorry it's a bit of reversion 😞 )